doajtest/unit/test_harvester.py

```diff
@@ -0,0 +1,97 @@
+import itertools
+import json
+import time
+from datetime import date, datetime
+
+from unittest.mock import Mock, patch
+
+from doajtest.fixtures import AccountFixtureFactory, JournalFixtureFactory, ArticleFixtureFactory
+from doajtest.fixtures.harvester import HarvestStateFactory, EPMCFixtureFactory
+from doajtest.helpers import DoajTestCase
+from portality.api.v2.client.client import DOAJv1API
+from portality.core import app
+from portality.tasks.harvester import HarvesterBackgroundTask
+from portality import models
+from portality.tasks.harvester_helpers.epmc import models as h_models
+from portality.lib import httputil
+from portality.api.v2.client import client as doajclient
+from portality.tasks.harvester_helpers import workflow
+from portality.tasks.harvester_helpers.workflow import HarvesterWorkflow
+from portality.models.harvester import HarvestState
+from portality.models.harvester import HarvesterPlugin
+
+
+class TestHarvester(DoajTestCase):
+
+
+    def mock_get_journals_issns(self, string1):
+        return ["1234-5678", "9876-5432"]
```

Comment:
I think we decided that this was no longer necessary, and we should handle it by injecting the correct data into the database instead

Reply:
Yes, it is unnecessary